lmdeploy/pytorch/engine/executor/base.py

```diff
@@ -89,6 +91,20 @@ async def forward_async(self, inputs):
     async def get_output_async(self):
         """get output async."""
         raise NotImplementedError('Not Implemented')
+    
+    """ PD Disaggregation API Begin """
+    def p2p_initialize(self, remote_engine_id: int, remote_engine_config: DistServeEngineConfig):
```

Comment:
the arguments of this method are not aligned with that in ray_executor
https://github.com/InternLM/lmdeploy/blob/b0a8f1fc8b157be0b235b6f18707c13ee8c36f16/lmdeploy/pytorch/engine/executor/ray_executor.py#L530